app/javascript/components/users/user/Avatar.jsx

```diff
@@ -19,9 +19,10 @@ export default function Avatar({ avatar, radius, className }) {
         width={radius}
         height={radius}
         className={className}
+        style={{ display: isLoaded ? '' : 'none' }}
```

Comment:
Essentially, the avatar was still taking up space while it was loading, which was causing the visual glitch. This change will make the avatar invisible until it loads in (replacing the placeholder)

Reply:
Interesting